packages/vats/src/bootstrap.js

```diff
@@ -29,6 +29,10 @@ const QUOTE_INTERVAL = 5 * 60;
 
 const BASIS_POINTS_DENOM = 10000n;
 
+// Minimum number of decimal places to consider before publishing a notifier
+// update from the bank.
+const BANK_BALANCE_UPDATE_DECIMAL_PLACES = 2;
```

Comment:
So if 1 RUN (1,000,000 uRUN) is around $1 in value, this creates a threshold of around 1 cent, which seems appropriate. But if other assets such as BLD increase in value, this may have unexpected consequences. For instance, if BLD is $40 each, then by my calculations it's about 40 cents, which is still appropriate, but if it's say 400 dollars, then that's 4 dollars as a threshold. Long term, I think this is something that we need to be able to set with governance on a per asset basis rather than hard-code. Short term this seems fine.

Reply:
I'm making a change to allow the thresholds to be configured over time, such as via governance.  The hardcoded amounts in bootstrap.js for `ubld` and `urun` are waiting for a revamp of the bootstrap to use governance pervasively.

Reply:
Awesome!